home-page/style.css

```diff
@@ -38,6 +39,7 @@ body::-webkit-scrollbar-thumb {
 .header {
   background-color: #900e78;
   color: white;
+  padding-top: 10px;
```

Comment:
This improves the look a lot.  Thanks for catching it.

Reply:
Welcome